MANIFEST.SKIP

```diff
@@ -17,7 +17,6 @@
 ^parts/base-
 ^ppport\.h$
 ^PPPort\.c$
-^PPPort.pm$
```

Comment:
we are going to ship one generated `PPPort.pm` file in the tarball so metacpan could use it for documentation